forms-flow-web/src/services/PdfService.js

```diff
@@ -22,13 +22,18 @@ export const exportToPdf = ({formId, formName, pdfName}) => {
 };
 
 export const printToPDF = ({ pdfName, formName }) => {
+  // hide block with class .hidden-in-print
+  const hiddenInPrint = document.querySelectorAll(".hidden-in-print");
+  hiddenInPrint.forEach((btmElm) => (btmElm.style.display = "none"));
   // Hiding the floating buttons during the PDF generation
-  const floatingButtons = document.querySelectorAll("button.floatingButton");
+  const selectors = "button.floatingButton,.formio-component-button";
+  const floatingButtons = document.querySelectorAll(selectors);
   floatingButtons.forEach((btmElm) => (btmElm.style.visibility = "hidden"));
   toast.success("Downloading...");
   exportToPdf({ formId: "formview", pdfName, formName });
   setTimeout(() => {
     floatingButtons.forEach((btmElm) => (btmElm.style.visibility = "visible"));
+    hiddenInPrint.forEach((btmElm) => (btmElm.style.display = "block"));
```

Comment:
Can we ensure these elements have had "block" for their display? If not, maybe you can save their display types before setting them to "none" and restore them here.

Reply:
Yeah! that's a great catch. That must be the case.
I have updated the code with the condition. Hide visible complement, keep a track of them and make only those visible again.